source/developers-guide/plugin-quick-start/index.md

```diff
@@ -24,11 +24,13 @@ On the left you can see Shopware's default directory structure, as you will find
 
 **build**: (Not in release packages) Contains our ant-based build engine with various scripts to e.g. install dependencies, reset the database etc. When installing Shopware from GitHub you want to run `ant build-unit` first. If you use the install package, you can just use our web-based installer
 
+**custom/plugins** This directory contains all plugin for shopware 5.2 and above.
```

Comment:
This directory contains all plugins based on the plugin system introduced with Shopware 5.2

Reply:
d